windows-driver-docs-pr/install/adding-software-components-with-an-inf-file.md

```diff
@@ -23,11 +23,13 @@ This page provides guidelines for the use of software components.
 
 A software component is a separate, standalone driver package that references one or more software modules.  The referenced software enhances the value of the device, but is not necessary for basic device functionality and has no associated function driver service.  
```

Comment:
The phrase "references one or more software modules" isn't strictly true, is that the story we want to display?  Or make it clear that a Software Component need not use AddSoftware?